bitnami/thanos/templates/serviceaccount.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ include "thanos.serviceAccountName" . }}
+  name: {{ include "thanos.serviceAccount.name" . }}
```

Comment:
Why don't we follow the same approach we have in others here?

https://github.com/bitnami/charts/blob/main/bitnami/wordpress/templates/_helpers.tpl#L50

Reply:
Done!